### PR TITLE
feat(ST-2967): add new company member. Set password page

### PR DESCRIPTION
**Pull request by trueboroda**
Description:

--

QA-test:

Demo-test:

Download artifact URL:

### CODE REVIEW

File: client-app/xapi/graphql/account/mutations/resetPasswordByToken/resetPasswordByToken.graphql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+mutation resetPassword($command: InputResetPasswordByTokenType) {
```
Comment (artmakarov) [inline client-app/xapi/graphql/account/mutations/resetPasswordByToken/resetPasswordByToken.graphql:1]: rename `ResetPasswordByToken`

File: client-app/xapi/graphql/account/index.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ export { default as updateMemberAddresses } from "./mutations/updateMemberAddres
 export { default as updatePersonalData } from "./mutations/updatePersonalData";
 export { default as createUser } from "./mutations/createUser";
 export { default as createContact } from "./mutations/createContact";
+export { default as resetPasswordByToken } from "./mutations/resetPasswordByToken";
```
Comment (artmakarov) [inline client-app/xapi/graphql/account/index.ts:12]: add `requestPasswordReset` here

File: client-app/shared/account/composables/useUser.ts [hunk 3]
```diff
@@ -15,6 +15,7 @@ import {
   ValidateToken,
 } from "@/shared/account";
 import globals from "@/core/globals";
+import requestPasswordReset from "@/xapi/graphql/account/queries/requestPasswordReset";
```
Comment (artmakarov) [inline client-app/shared/account/composables/useUser.ts:18]: add import from `@/xapi/graphql/account` in line 4

File: client-app/shared/account/composables/useUser.ts [hunk 4]
```diff
@@ -160,26 +161,18 @@ export default () => {
...
-      return await innerFetch<IdentityResultType, ValidateToken>(url, "POST", payload);
+      const success = await requestPasswordReset({
+        loginOrEmail: payload.email,
+        urlSuffix: payload.resetPasswordUrl,
+      });
+
+      return success;
     } catch (e) {
-      Logger.error("useUser.validateToken", e);
+      Logger.error("useUser.registerUser", e);
```
Comment (artmakarov) [inline client-app/shared/account/composables/useUser.ts:175]: `Logger.error('useUser.${forgotPassword.name}', e);`

**Review by artmakarov — APPROVED** [review]

**Review by Andrew-Orlov — APPROVED** [review]

File: client-app/xapi/graphql/account/index.ts (hunk 2 above)
Comment (trueboroda) [reply client-app/xapi/graphql/account/index.ts:12]: it's already added. It is query but not mutation.

**Pull request merged**